lib/src/RedisClientSkipped.cc

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ *  RedisClientSkipped.cc
+ *  An Tao
+ *
+ *  Copyright 2018, An Tao.  All rights reserved.
+ *  https://github.com/an-tao/drogon
+ *  Use of this source code is governed by a MIT license
+ *  that can be found in the License file.
+ *
+ *  Drogon
+ *
+ */
+
+#include "drogon/nosql/RedisClient.h"
+
+namespace drogon
+{
+namespace nosql
+{
+std::shared_ptr<RedisClient> RedisClient::newRedisClient(
+    const trantor::InetAddress &serverAddress,
+    size_t numberOfConnections,
+    const std::string &password)
+{
+    LOG_FATAL << "Redis is supported by drogon, please install the "
```

Comment:
Redis is not supported by drogon ...

Reply:
Just copied from `RedisClientManagerSkipped.cc` I'll correct there too

Reply:
Thanks :)